Makefile

```diff
@@ -1,5 +1,6 @@
 export RELEASE_BRANCH?=1-18
-export RELEASE?=1
+export PULL_NUMBER?=1
```

Comment:
@TerryHowe  👋   i would suggest maybe adding few comments and maybe provide a real example so folks who are not AWS employees and may miss internal chats/ context will get it the meaning. 

Otherwise folks will have to go right to the [main script](https://github.com/aws/eks-distro/pull/137/files#diff-cd47c42ec890bee60a3b95dfea693761b7dd991ef47ab10a32ce99aaf32ef08bR18-R20)  .

Reply:
Fair point @DanyC97 it makes me think several things
* I hadn't gotten around to even putting a description on this PR yet
*  The `PULL_NUMBER` is really just a build number and probably should not override RELEASE. I think it would be better adding it after the release number like 1.18.1.137
* The pull number is just for build releases. Production releases will go out like 1.18.1
* The pull number environment variable is supplied by Prow and will match the pull request #137 in this case

Reply:
The other thing is I think the only internal chats here are in my head :)

Reply:
thanks for the info @TerryHowe , it does clarify things indeed. 👍